client/app/components/game-client.js

```diff
@@ -0,0 +1,62 @@
+import Ember from 'ember';
+import GameChannel from 'vilive/utils/game-channel';
+
+export default Ember.Component.extend({
+  init() {
+    this.set('messages', Ember.makeArray());
+    GameChannel.init(this.get('session-manager').currentToken());
+    GameChannel.on('event', payload => {
+      this.putMessage({
+        server: {
```

Comment:
Playing Achaea it was clear that there is need for a markup of some kind we translate into html for the ember stuff and other clients digest as well. 

I do wonder if markdown would not be a good place to start. For example noticed in Achaea the inventory command prints a table etc and markdown handles this.

I'd imagine that there libs as well to translate markdown into ascii as well. Not sure though but can't help people have extended and repurposed markdown. 

But yeah right now was going for somethings simple to get us started. 

Sent from my iPhone

> On Jan 16, 2016, at 3:01 PM, Sean Copenhaver <notifications@github.com> wrote:
> 
> In client/app/components/game-client.js:
> 
> > @@ -0,0 +1,62 @@
> > +import Ember from 'ember';
> > +import GameChannel from 'vilive/utils/game-channel';
> > +
> > +export default Ember.Component.extend({
> > +  init() {
> > +    this.set('messages', Ember.makeArray());
> > +    GameChannel.init(this.get('session-manager').currentToken());
> > +    GameChannel.on('event', payload => {
> > +      this.putMessage({
> > +        server: {
> I fully expect us to have to "render" the message in code some how. I would think we would want to wrap item names, player names, and NPC names with <span> and different classes to change the styling or add links for quick commands. These are all much later concerns though since we'll have to actually have a game working before we can experiment with message structure and rendering.
> 
> —
> Reply to this email directly or view it on GitHub.
>